### ISSUE TITLE
Hyperparameter Tuning - Step 6

### ISSUE THREAD

**Issue by IKEA0x90** [evil-cry/cyber-analytics-ml#21]
Consider the following parameters for tuning:
- value C
- number of iterations
- learning rate
<b>Find appropriate techniques or use bruteforce again?</b>

**Comment by evil-cry** [comment]
https://github.com/evil-cry/cyber-analytics-ml/pull/31
